core/shared/src/main/scala/laika/directive/api.scala

```diff
@@ -259,7 +260,7 @@ trait BuilderContext[E <: Element] {
         .asInstanceOf[E]
     }
 
-    def runsIn(phase: RewritePhase): Boolean = directive.fold(true)(_.runsIn(phase))
+    def runsIn(@nowarn phase: RewritePhase): Boolean = directive.fold(true)(_.runsIn(phase))
```

Comment:
Another option here is to `import org.typelevel.scalaccompat.annotation.unused` (sbt-typelevel includes the necessary dependency by default). It's a bit more specific than a generic `@nowarn`.

```suggestion
    def runsIn(@unused phase: RewritePhase): Boolean = directive.fold(true)(_.runsIn(phase))
```

Reply:
yup, that's nicer